PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -762,6 +931,21 @@ public IEnumerable<CallStack> GetCallStack()
                             (string)psFrame.FunctionName.ToString(),
                             (int)psFrame.ScriptLineNumber));
                 }
+                else if(GetDebugScenario() == DebugScenario.Local_Attach)
+                {
+                    // local process attach debugging
+                    string currentCall = psobj.ToString();
+                    Match match = validStackLine.Match(currentCall);
+                    if (match.Success)
+                    {
+                        String funcall = match.Groups[1].Value;
+                        String script = match.Groups[3].Value;
+                        int lineNum = int.Parse(match.Groups[4].Value);
+
+                        callStackFrames.Add(new CallStack(script, funcall, lineNum));
+                    }
+                }
+

```

Comment:
Do we want a Debug.Assert, or some other error handling that the DebugScenario isn't unknown, as it seems we expect at least some known scenario here?